app/styles/prose.css

```
@@ -899,14 +899,52 @@ pre {
+	color: var(--text-primary);
+}
+
+.prose :where(form, .demo) :where(input, select, textarea):not(
+		[type='checkbox']
+	):not([type='radio']) {
+	background-color: var(--bg-primary);
+}
+
+.prose :where(form, .demo) :where(button):not([type='checkbox']):not(
+		[type='radio']
+	) {
```

Stylelint (17.3.0)

[error] Expected complex :not() pseudo-class notation (selector-not-notation)

(selector-not-notation)